docs/dev/apis/configurable-company/developing-categories.md

```diff
@@ -0,0 +1,71 @@
+---
+prev: true
+next: true
+description: How to create categories for your plugin and how to use them.
```

Comment:
add 'Configurable Company' to this
...create Configurable Company categories...